import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/backupuploader/BackupUploadService.java

```diff
@@ -100,25 +110,36 @@ private void scheduleUploadWithDelay(IDicomWebClient webClient, BackupState back
         throw new IBackupUploader.BackupException(eex.getCause());
       }
     } else {
-      throw getNoResendAttemptLeftException(uniqueFileName);
+      throw getNoResendAttemptLeftException(null, uniqueFileName);
```

Comment:
Can we pass along the original exception message in No Resend attempts left and throwOnHttpFilterFail so that no details are lost?

Reply:
We do not miss anything. In normal workflow this check is never happens, because we have outer check 
attempts > 0 in CStoreService. This check presented like an double check for future usage/reusage of BackupUploadService.
I can remove it if you want.

Reply:
I see, yes you are right for this particular exception, and keeping it as-is is fine.

But for the calls to 
throwOnNoResendAttemptsLeft(dicomStatus, uniqueFileName);
and 
throwOnHttpFilterFail(dicomStatus, uniqueFileName);
above, within the CompletableFuture.Async call,  it would be nice to include the original exception stacktrace in these messages.